### ISSUE TITLE
About academic dataset 

### ISSUE THREAD

**Issue by rootlu** [luckiezhou/DynamicTriad#11]
I analyzed the published data set, the number of sides is only 714470, which is inconsistent with the paper, is it different from the used dataset in the paper?

**Comment by rootlu** [comment]
Another question: 

The id of nodes in the datasets is not continuous (max id: 2024483), so how to assign the label to a node?

Do you sort the ids from small to large and then assign labels in turn?

**Comment by luckiezhou** [comment]
The number of edges reported in the paper is not counted on static graphs, but counted on the graphs built under our time step configuration.
16 graphs are generated from the original static graphs in our paper, and each of them is the combination (summing up the weights for the same edge) of 4 consecutive static graphs.
Specifically, the 16 graphs are built from static graphs 0-3, 2-5, 4-9, ..., 30-33.
Using our script, the sequence of graphs are generated by setting "-l 4 -s 2 -n 16".

**Comment by skx300** [comment]
Hi Zhou, why there is an overlapping between each time step? While for the other two datasets, Mobile and Loan, there is no overlapping.

**Comment by luckiezhou** [comment]
It is not guaranteed that each researcher, especially graduate students, publishes at least one paper per year. In order to build a more temporally stable dynamic network, I have to merge the records in a short period (4 years considering the average time a Ph.D. student focuses on publications) to build the graph of a single timestep.
I think the ideal value for the stride is 1 (i.e. a 3-year-overlap) so that each student-mentor link may appear in several consecutive time steps, however, I choose 2 simply because there are a moderate number of time steps generated under this setting.

**Comment by skx300** [comment]
Thank you Zhou. Does it mean all vertices should appear in every time snapshot?

**Comment by luckiezhou** [comment]
Yes. All the snapshots share the same vertex set, although some vertices might be isolated in certain snapshots.
